### PR TITLE
Created initial show_drops UI Flask templates

### PR DESCRIPTION
**Pull request by AlexTryjan**
Setup Flask, Created show_drops template with ability to view dummy drop info & perform subscriber commands using buttons

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# frontend
\ No newline at end of file
+# frontend
+# Setup Virtual Environment: virtualenv venv
```
Comment (matthewbentley) [inline README.md]: nit: make this a numbered list; in markdown '#' indicates a H1 block

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,2 @@
+graft frontend/templates
```
Comment (DLance96) [inline MANIFEST.in]: Is MANIFEST from Flask or your IDE, if IDE remove this file

Comment (AlexTryjan) [reply MANIFEST.in]: Flask required file

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+setup(
+    name='frontend',
+    packages=['frontend'],
```
Comment (matthewbentley) [inline setup.py]: 'syncr_frontend' here

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+setup(
+    name='frontend',
```
Comment (matthewbentley) [inline setup.py]: '5yncr Frontend'

File: setup.py (hunk 3 above)
Comment (DLance96) [inline setup.py]: setup(
    name='5yncr Tracker',
    version='0.0.1',
    packages=['syncr_tracker', ],
    license='AGPLv3',
)

Your's should match naming conventions of backend and tracker

File: MANIFEST.in (hunk 2 above)
Comment (DLance96) [reply MANIFEST.in]: 🐒

**Comment by AlexTryjan** [comment]
@DLance96 @matthewbentley

**Pull request merged**